analyzer/src/main/kotlin/PackageManager.kt

```diff
@@ -119,7 +123,8 @@ abstract class PackageManager(val projectType: String) : Plugin {
                 }
             }.filter { it.isDirectory }.forEach { dir ->
                 val filesInCurrentDir = dir.walk().maxDepth(1).filterTo(mutableListOf()) {
-                    it.isFile && !excludes.isPathExcluded(rootPath, it.toPath())
+                    val isIncluded = includes.isPathIncluded(rootPath, it.toPath()) || includes == Includes.EMPTY
```

Comment:
The `includes == Includes.EMPTY` check is risky if scope includes are added later. I think it would be better if `isPathIncluded` would return `true` if no paths are configured as suggested in a previous commit.